README.md

```diff
@@ -25,36 +25,40 @@ This project is [hosted on GitHub](https://github.com/lifeomic/phc-sdk-py). You
 git clone git@github.com:lifeomic/phc-sdk-py.git
 ```
 
-### Development
+## Development
 
-Python environments are managed using [virtualenv](https://virtualenv.pypa.io/en/latest/). Be sure to have this installed first `pip install virtualenv`. The makefile will setup the environment for the targets listed below.
+### Prerequisite
 
-Some clients in this SDK are auto-generated. They can be re-generated at any time to pull in the latest changes by running `make gen`.
```

Comment:
I'd be nice to still keep documentation here that describes running `poetry run poe gen` to update the auto-generated portions of the SDK.

Reply:
Poetry will use the `version` field in the `pyproject.toml` file to decide what version to release the package under, so we should update the "Release Process" section down below as well to describe that. And we can get rid of the `phc.version` file.

Reply:
let me added it back.

Reply:
good point on versioning. let me change it in another commit

Reply:
see https://github.com/lifeomic/phc-sdk-py/pull/189/commits/eae0227864d23e81cefa1bca43c64ec069e5f57b